src/chat_completion_request.h

```diff
@@ -28,6 +48,24 @@ inline ChatCompletionRequest fromJson(std::shared_ptr<Json::Value> jsonBody) {
     completion.messages = (*jsonBody)["messages"];
     completion.stop = (*jsonBody)["stop"];
     completion.model_id = (*jsonBody).get("model", {}).asString();
+
+    completion.seed = (*jsonBody).get("seed", -1).asInt();
```

Comment:
I notice this PR defines default values twice:
- struct definition (above)
- JSON parsing default value

Are we able to define once?

DRY principle: https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

Reply:
I followed the previous implementation like this one https://github.com/janhq/cortex.llamacpp/blob/main/src/chat_completion_request.h#L8, maybe some weird bug  in the past force us to do it. Like this [PR](https://github.com/janhq/cortex.llamacpp/pull/181/files) I fixed the race condition, even if we checked the everything code (mutex, only return slot if available, ... ) but error still pop up. So I have to add another check for `if (slot==null)` so that the issue can be resolved. 

We are using third party lib for `json` so I think it's no harm, but if it necessary, I'll change it, but we need to test more. to make sure it won't break any thing